README.md

```diff
@@ -167,23 +50,17 @@ See [Forking](./docs/forking.md) for more information about creating your own fo
 
 ## Contributing
 
-See [CONTRIBUTING.md](CONTRIBUTING.md)
-
-We meet weekly Monday's at 8:15 AM PT. The meeting is subject to change
-depending on contributors' availability. Check the [OpenTelemetry community
-calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)
-for specific dates and Zoom meeting links.
+To get involved with the project see our [CONTRIBUTING](CONTRIBUTING.md)
+documentation. Our [SIG Calls](CONTRIBUTING.md#join-a-sig-call) are Mondays at
+8:15 AM PST and anyone is welcome.
 
-Meeting notes are available as a public [Google
-doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98/edit).
-For edit access, get in touch on
-[Slack](https://cloud-native.slack.com/archives/C03B4CWV4DA).
+## Project leadership
 
 [Maintainers](https://github.com/open-telemetry/community/blob/main/community-membership.md#maintainer)
 ([@open-telemetry/demo-maintainers](https://github.com/orgs/open-telemetry/teams/demo-maintainers)):
 
 - [Austin Parker](https://github.com/austinlparker), Lightstep
-- [Carter Socha](https://github.com/cartersocha), Microsoft
+- [Carter Socha](https://github.com/cartersocha), Lightstep
```

Comment:
this is not a readme refactor 🤣

Reply:
Lol not really but I figured while I was making the other major changes